lib/core/widgets/home_page_app_bar.dart

```diff
@@ -547,7 +547,7 @@ class DeleteIcon extends StatelessWidget {
                     mainAxisSize: MainAxisSize.min,
                     children: [
                       Text(
-                        "You are about to delete $deletionCount  item${deletionCount > 1 ? "s" : ""}",
+                        "${S.current.aboutToDelete} $deletionCount ${S.current.item}${deletionCount > 1 ? "s" : ""}",
                         style: TextStyle(
```

Comment:
`${S.current.item}${deletionCount > 1 ? "s" : ""}",` this part wouldn't be meaningful for other languages. The purpose of this was to display `item` if the count is 1 and `items` if the count > 1, but in other languages it won't be `s` at the end for plural of item. I'll think about how to handle this, let me know if you have any solution

Reply:
Hi, let's keep two different strings for `item` and `items`

`${S.current.aboutToDelete} $deletionCount ${deletionCount > 1 ? S.current.item : S.current.items}`

Reply:
yeah I agreed